package.json

```diff
@@ -46,7 +46,9 @@
     ],
     "nohoist": [
       "**/react-codemirror2",
-      "**/react-codemirror2/**"
+      "**/react-codemirror2/**",
+      "app/webpack",
+      "app/acorn-dynamic-import"
```

Comment:
(Hacky?) Fix for https://github.com/webpack/webpack/issues/8656 until https://github.com/acornjs/acorn/pull/834 gets merges.